app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> str | None:
+    if not command.startswith("mv "):
+        return "Wrong command"
+
+    _, source_file, new_file = command.split()
```

Comment:
Make sure that command consists of exactly three elements before split or use
```suggestion
    _, source_file, new_file, *_ = command.split()
```